### PR TITLE
Add testId to carousel element

### PR DESCRIPTION
**Pull request by VolhaHurynovich**
iOs:

iosItemName2

Android:

androidItemSlide2

### CODE REVIEW

File: src/screens/CarouselScreen.js [hunk 1]
```diff
@@ -13,22 +13,27 @@ class CarouselScreen extends Component {
         {
           title: 'Item 1',
           text: 'Text 1',
+          itemName: 'name_1',
```
Comment (ikalinin1) [inline src/screens/CarouselScreen.js:16]:
```suggestion
          itemName: 'item 1',
```

Comment (ikalinin1) [reply src/screens/CarouselScreen.js:16]: ?

Comment (VolhaHurynovich) [reply src/screens/CarouselScreen.js]: changed to `slide_1`

**Review by ikalinin1 — APPROVED** [review]

File: .github/workflows/ios-ci.yml [hunk 2]
```diff
@@ -38,9 +38,6 @@ jobs:
       - name: Run test
         run: npm run test
 
-      - name: Select Xcode 12.4
-        run: sudo xcode-select --switch /Applications/Xcode_12.4.app
-
```
Comment (valfirst) [inline .github/workflows/ios-ci.yml:43]: the changes must be atomic

File: src/screens/CarouselScreen.js [hunk 3]
```diff
@@ -13,22 +13,27 @@ class CarouselScreen extends Component {
         {
           title: 'Item 1',
           text: 'Text 1',
+          itemName: 'slide_1',
```
Comment (valfirst) [inline src/screens/CarouselScreen.js:16]:
```suggestion
          itemName: 'carousel_item_1',
```
and so on

File: .github/workflows/ios-ci.yml (hunk 2 above)
Comment (VolhaHurynovich) [reply .github/workflows/ios-ci.yml:43]: done

File: src/screens/CarouselScreen.js (hunk 3 above)
Comment (VolhaHurynovich) [reply src/screens/CarouselScreen.js]: done

File: src/screens/CarouselScreen.js [hunk 4]
```diff
@@ -37,6 +42,7 @@ class CarouselScreen extends Component {
   _renderItem({item, index}) {
     return (
       <View
+        testID={item.itemName}
```
Comment (valfirst) [inline src/screens/CarouselScreen.js:45]: it makes sense to use `utils/automationIDs` to generate cross-platform IDs

Comment (VolhaHurynovich) [reply src/screens/CarouselScreen.js]: done

File: src/screens/CarouselScreen.js [hunk 5]
```diff
@@ -13,30 +13,35 @@ class CarouselScreen extends Component {
...
         },
       ],
     };
   }
 
   _renderItem({item, index}) {
     return (
-      <View
+      <View {...automationIDs(item.itemName)}
         style={{
```
Comment (valfirst) [inline src/screens/CarouselScreen.js:45]:
```suggestion
      <View 
        {...automationIDs(item.itemName)}
        style={{
```

Comment (VolhaHurynovich) [reply src/screens/CarouselScreen.js]: thank you, done

**Review by valfirst — APPROVED** [review]

**Pull request merged**